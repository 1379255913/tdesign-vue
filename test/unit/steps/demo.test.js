import { mount } from '@vue/test-utils';

// unit test for component in examples.
describe('Steps', () => {
  it('base demo works fine', () => {
    const demo = require('@/examples/steps/demos/base.vue');
    const wrapper = mount(demo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
