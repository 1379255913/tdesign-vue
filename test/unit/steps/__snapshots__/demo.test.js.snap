// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Steps Steps NoSequence demo works fine 1`] = `
<div
  class="t-steps t-steps--horizontal t-steps--dot-anchor"
/>
`;

exports[`Steps Steps VerticalNoSequence demo works fine 1`] = `
<div
  class="setps-demo"
>
  <div
    class="tdesign-demo-block"
  >
    <div
      class="t-steps t-steps--vertical t-steps--dot-anchor t-steps--positive"
    />
     
    <div
      class="t-steps t-steps--vertical t-steps--dot-anchor t-steps--reverse"
    />
  </div>
</div>
`;

exports[`Steps Steps VerticalSequence demo works fine 1`] = `
<div
  class="setps-demo"
>
  <div
    class="tdesign-demo-block"
  >
    <div
      class="t-steps t-steps--vertical t-steps--default-anchor t-steps--positive"
    />
     
    <div
      style="margin: 16px 0px;"
    />
     
    <div
      class="t-steps t-steps--vertical t-steps--default-anchor t-steps--reverse"
    />
  </div>
</div>
`;

exports[`Steps Steps extra demo works fine 1`] = `
<div
  class="t-steps t-steps--vertical t-steps--default-anchor t-steps--positive"
/>
`;

exports[`Steps Steps icon demo works fine 1`] = `
<div
  class="t-steps t-steps--horizontal t-steps--default-anchor"
/>
`;

exports[`Steps Steps options demo works fine 1`] = `
<div
  class="t-steps t-steps--horizontal t-steps--default-anchor"
/>
`;

exports[`Steps Steps sequence demo works fine 1`] = `
<div>
  <div
    class="t-steps t-steps--horizontal t-steps--default-anchor"
  />
   
  <br />
   
  <div
    class="t-steps t-steps--horizontal t-steps--default-anchor"
  />
</div>
`;

exports[`Steps Steps status demo works fine 1`] = `
<div>
  <div
    class="t-steps t-steps--horizontal t-steps--default-anchor"
  />
   
  <div
    style="margin: 16px 0px;"
  />
   
  <div
    class="t-steps t-steps--horizontal t-steps--default-anchor"
  />
</div>
`;
